Dockerfile

```diff
@@ -75,6 +91,7 @@ RUN  yum -y install --disableplugin=subscription-manager \
      && yum --disableplugin=subscription-manager clean all
 COPY --from=build-stage0 /out/oc  /usr/local/bin
 COPY --from=build-stage0 /out/oc-hc  /usr/local/bin
+COPY --from=build-stage0 /out/yq_linux_amd64  /usr/local/bin
```

Comment:
do we want to rename this to `yq`?

Reply:
It's not needed, but I renamed it for clean binary name